lib/clients/AzureClient.py

```diff
@@ -126,6 +130,17 @@ def get_host_number_of_data_volumes(self):
                 'of persistent volume directory {}.{}'.format(self.DIRECTORY_PERSISTENT, error))
         return host_number
 
+    def get_instance_location(self, instance_id):
+        try:
+            instance = self.compute_client.virtual_machines.get(
+                self.resource_group, instance_id)
+            return instance.location
+        except Exception as error:
+            self.logger.error(
+                '[Azure] ERROR: Unable to find or location for instance_id {}.{}'.format(
```

Comment:
Please check the error statement. Maybe you can only mention "Unable to get location".

Reply:
done